### PR TITLE
[feature/v1-054] Added new Sign in form style, response design

### PR DESCRIPTION
**Pull request by NikitaKhadnevich**
**Now this pull request include next features:** 

1. Added new fonts for Sign In page;
2. Recreated form and added new "pixel perfect" design style;
3. Implemented `Formik Error` feature;
4. Updated `breakpoints`;
5. Added new responsive style 320 - 1440+ resolution.

### CODE REVIEW

File: frontend/src/api/logOut/logOut.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useMutation, UseMutationResult } from 'react-query';
+import { useNavigate } from 'react-router';
+
+import { apiClientWrapper } from 'api/base';
+import { API, PATHS } from 'constants/routes';
+import { REQUEST_ERRORS } from 'constants/authConstants';
+
+import { logOutHandler } from 'utils/helpers/logOutHandler';
```
Comment (hauteballe) [inline frontend/src/api/logOut/logOut.tsx:8]: extra empty line

File: frontend/src/components/TextField/TextField.tsx [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
 import { TextField as MuiTextField } from '@mui/material';
-import { Field } from 'formik';
+import { Field, ErrorMessage } from 'formik';
+
+import WarningHelper from './styled';
 
 interface TextFieldProps {
   id: string;
-  label: string;
+  label?: string;
```
Comment (hauteballe) [inline frontend/src/components/TextField/TextField.tsx:10]: Please sort all the props by their requirement, at first required and  then possible

File: frontend/src/components/TextField/TextField.tsx [hunk 3]
```diff
@@ -21,29 +31,34 @@ const TextField: React.FC<TextFieldProps> = ({
...
-      error={touched && Boolean(error)}
-      {...rest}
-    >
-      {children}
-    </Field>
-    {error}
-  </>
-);
+}) => {
+  return (
```
Comment (hauteballe) [inline frontend/src/components/TextField/TextField.tsx:42]: Please remove return statement here

File: frontend/src/pages/SignIn/Definition/Definition.tsx [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 const Definition = (): JSX.Element => (
   <Wrapper>
     <LogoBox>
-      <CompanyLogo>:iTechArt LogIn</CompanyLogo>
+      <CompanyLogo>
+        <span>:i</span>Tech<span>Art</span>
+      </CompanyLogo>
     </LogoBox>
-    <Box sx={{ mt: 2 }}>
+    <LogoBox sx={{ mt: 2 }}>
```
Comment (hauteballe) [inline frontend/src/pages/SignIn/Definition/Definition.tsx:12]: Move to styled.ts please

**Review by hauteballe — CHANGES_REQUESTED** [review]

File: frontend/src/pages/SignIn/SignIn.tsx [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
 import Button from 'components/Button';
 import TextField from 'components/TextField';
 import { SignTypes } from 'types/signIn';
+import { signInImage } from 'icons';
 
+import Image from 'components/Image';
 import Definition from './Definition';
+
 import {
   SignWrapper,
```
Comment (Julia-Maliutsina) [inline frontend/src/pages/SignIn/SignIn.tsx:12]: import { SignTypes } from 'types/signIn';
import { signInImage } from 'icons';
import Image from 'components/Image';

import Definition from './Definition';
import {  SignWrapper, ... } from './styled.ts'

**Review by Julia-Maliutsina — CHANGES_REQUESTED** [review]

File: frontend/src/api/logOut/logOut.tsx (hunk 1 above)
Comment (NikitaKhadnevich) [reply frontend/src/api/logOut/logOut.tsx:8]: Fixed

File: frontend/src/components/TextField/TextField.tsx (hunk 2 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/TextField/TextField.tsx:10]: Sorted these lines

File: frontend/src/components/TextField/TextField.tsx (hunk 3 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/TextField/TextField.tsx:42]: Done

File: frontend/src/pages/SignIn/Definition/Definition.tsx (hunk 4 above)
Comment (NikitaKhadnevich) [reply frontend/src/pages/SignIn/Definition/Definition.tsx:12]: Moved

File: frontend/src/pages/SignIn/SignIn.tsx (hunk 5 above)
Comment (NikitaKhadnevich) [reply frontend/src/pages/SignIn/SignIn.tsx:12]: FIxed

File: frontend/src/pages/SignIn/SignIn.tsx [hunk 6]
```diff
@@ -2,12 +2,12 @@ import React from 'react';
 
 import Button from 'components/Button';
 import TextField from 'components/TextField';
+
 import { SignTypes } from 'types/signIn';
```
Comment (hauteballe) [inline frontend/src/pages/SignIn/SignIn.tsx:6]: extra empty line again

**Review by hauteballe — CHANGES_REQUESTED** [review]

File: frontend/src/pages/SignIn/SignIn.tsx (hunk 6 above)
Comment (NikitaKhadnevich) [reply frontend/src/pages/SignIn/SignIn.tsx:6]: fixed

File: frontend/src/components/TextField/TextField.tsx [hunk 7]
```diff
@@ -1,48 +1,67 @@
...
+  handleBlur,
   ...rest
 }) => (
   <>
     <Field
       component={MuiTextField}
-      onChange={onChange}
+      onChange={(e: string) => {
+        handleChange(e);
+        setFieldTouched(id, isFieldTouched);
```
Comment (k-krasilnikova) [inline frontend/src/components/TextField/TextField.tsx:50]: Move it please inside separate common function and call it here:
```suggestion
        onChange={onChange}
```

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]
logout functionality is not working

File: frontend/src/components/TextField/TextField.tsx (hunk 7 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/TextField/TextField.tsx]: Created new function for this and filled it in Container

**Comment by NikitaKhadnevich** [comment]
Update. Implemented `warningHelper` function. Added logOut feature, based on mocked data

File: frontend/src/components/Layout/Header/Header.tsx [hunk 8]
```diff
@@ -98,7 +102,7 @@ const Header: React.FC<HeaderProps> = ({
             <UserName>{`${firstName} ${lastName}`}</UserName>
           </UserBlock>
         </NavLink>
-        <LogOut>
+        <LogOut onClick={() => mutateAsync({ login: 'user', password: 'user' })}>
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/Header/Header.tsx:105]: It's not the UI!

File: frontend/src/components/Layout/Header/Header.tsx [hunk 9]
```diff
@@ -49,6 +51,8 @@ const Header: React.FC<HeaderProps> = ({
   handleNotificationsOpen,
   handleFilterOpen,
 }) => {
+  const { mutateAsync } = useLogOut();
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/Header/Header.tsx:54]: It's not the UI!

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/components/Layout/Header/Header.tsx (hunk 8 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/Header/Header.tsx]: Removed this in main container

File: frontend/src/components/Layout/Header/Header.tsx (hunk 9 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/Header/Header.tsx]: Fixed this

File: frontend/src/components/Layout/Header/Header.tsx [hunk 10]
```diff
@@ -102,7 +106,7 @@ const Header: React.FC<HeaderProps> = ({
           <UserAvatar avatar={avatar} size="small" />
           <UserName>{`${firstName} ${lastName}`}</UserName>
         </UserBlock>
-        <LogOut>
+        <LogOut onClick={() => handleLogOut()}>
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/Header/Header.tsx:109]:
```suggestion
        <LogOut onClick={handleLogOut}>
```

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/components/Layout/Header/Header.tsx (hunk 10 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/Header/Header.tsx]: Fixed

**Review by hauteballe — CHANGES_REQUESTED** [review]
Screenshot at Jan 24 10-29-51

SignIn page has an additional scrollbar

File: frontend/src/components/Layout/Header/Header.tsx [hunk 11]
```diff
@@ -9,6 +10,7 @@ import { UserAvatar } from 'components/Avatar';
 import Logo from 'components/BrandLogo';
 import { alertIcon, filterIcon, logOutIcon } from 'icons';
 
+import { UseMutationResult } from 'react-query';
```
Comment (Julia-Maliutsina) [inline frontend/src/components/Layout/Header/Header.tsx:13]: should be at the top with other libraries imports

**Review by Julia-Maliutsina — CHANGES_REQUESTED** [review]

File: frontend/src/components/Layout/Header/Header.tsx (hunk 11 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/Header/Header.tsx]: Updated It

**Review by Julia-Maliutsina — APPROVED** [review]

**Pull request merged**